src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString
+    .split(';')
+    .map(el => el.trim())
+    .filter(element => element.length > 1)
+    .map(property => property.split(':')
+      .map(prop => prop.trim()))
+    .forEach((property) => {
```

Comment:
Краще використай метод reduce, щоб створити об'єкт і додати туди властивості

Reply:
Використай деструктуризацію
```suggestion
    .forEach(([key, value]) => {
```